models/ed/R/veg2model.ED2.R

```diff
@@ -0,0 +1,234 @@
+#' Writes ED specific IC files
+#'
+#' @param obs data either passed from upstram or NULL
+#' @param inputinfo
+#' @param runinfo run from settings
+#' @param outfolder where to write files
+#' @param host 
+#' @param con bety$con
+#' @param overwrite
+#' @return ic.id initial conditions file input id
+#' @export
+#' @author Istem Fer
+#'  We need the in.path, the out folder, the start and endates and the lat/long. All of these are in setting$run. 
+veg2model.ED2 <- function(obs = NULL, inputinfo, runinfo, outfolder, host, con, overwrite = FALSE){
+  
+  start_year <- lubridate::year(runinfo$start.date)
+  end_year   <- lubridate::year(runinfo$end.date)
+  startdate   <- lubridate::as_date(paste0(start_year, "-01-01"))
```

Comment:
I was copying the line from [fia2ED](https://github.com/PecanProject/pecan/blob/master/modules/data.land/R/fia2ED.R#L30_L31) to be honest, I also thought it was a weird thing to do but guessed it's an FIA quirk